Entity/BlockInteractor.php

```diff
@@ -160,6 +160,9 @@ public function loadPageBlocks(PageInterface $page)
         $page->disableBlockLazyLoading();
 
         foreach ($blocks as $block) {
+            if (!$block->getEnabled()) { 
```

Comment:
There is a trailing space at the end of the line, this is why StyleCI complains. Please remove it.